### ISSUE TITLE
핵을 사용하지않았는데 7명 이상의 유저한테 신고 먹고 벤을 당했습니다.

### ISSUE THREAD

**Issue by ssmedison** [Uuvana-Studios/longvinter-windows-client#675]
**Describe the bug**

**To Reproduce**
1. Go to broke the blue fence, but got a permanent ban on the official server.
There are other bugs, such as turret copying problem, password being unlocked even though the password is locked, NPC turret killing people, etc.

They killed people by using all these kinds of things, so they were mistaken for nuclear weapons.

https://youtu.be/2SoJ27WjWss

If you look at the video link above, it is clearly the blue area. So it was possible to unlock it, but I ate Ben by unlocking that.
Please check the log records.

The servers are daon 1,2,3 servers.

Please release VPN.'
4. See an error

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]
 - Game Version [e.g. 1.0]
 - Steam Version [e.g. 1.0]

**Additional context**

**Comment by faketucker** [comment]
Duplicate of #677

**Comment by ssmedison** [comment]
dear uuvana- studios 
hi im ssmdison, 
You said you would unblock me, but did it change? 
Thank you.
-----Original Message-----

**Comment by sirNugg3ts** [comment]
ssmedison, you have been flagged several times already with conclusive evidence, this was not the first time from what I have talked with the mods. Also you definitely crossed the line when emailing me to my personal mail about this issue. This ban will not be reverted. Please do not contact again trying to get unbanned
